### PR TITLE
LRAC-8176 Remove flags from quarantined and ignored tests

### PR DESCRIPTION
**Pull request by r4ddek**

### CODE REVIEW

**Comment by r4ddek** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
ci:test:sf

**Review by ViniLopes20 — APPROVED** [review]

**Review by manuferreira — APPROVED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsDocumentsAndMedia.testcase [hunk 1]
```diff
@@ -213,7 +211,6 @@ definition {
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8403 | Test Summary: Documents audience card shows expected amount of segmented and unsegmented individuals"
 	@priority = "5"
 	test AudienceCardShowsExpectedAmountSegmentedAndUnsegmentedInDM {
-		property portal.upstream = "quarantine";
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsDocumentsAndMedia.testcase:216]: Hi @r4ddek, could you keep this one in quarantine?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase [hunk 2]
```diff
@@ -41,7 +41,6 @@ definition {
 	@ignore = "true"
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase:41]: Hi @r4ddek, could you remove this from ignore?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase [hunk 3]
```diff
@@ -41,7 +41,6 @@ definition {
...
 	@priority = "5"
 	test AssertAppearsOnListShowsPagesFormsExists {
-		property portal.upstream = "quarantine";
 
 		// AC Refactor: It is not possible to perform the assert because this test presents timing problems.
 		// AC Refactor ticket: LRAC-9467
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase:46]: Hi @r4ddek, could you remove these lines?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase [hunk 4]
```diff
@@ -162,8 +159,6 @@ definition {
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8424 | Test Summary: Forms audience card shows expected amount of segmented and unsegmented individuals"
 	@priority = "5"
 	test AudienceCardShowsExpectedAmountSegmentedAndUnsegmentedInForms {
-		property portal.upstream = "quarantine";
-
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsForms.testcase:166]: Hi @r4ddek, could you keep this one in quarantine?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsWebContent.testcase [hunk 5]
```diff
@@ -296,7 +294,6 @@ definition {
 	@ignore = "true"
 	@priority = "4"
 	test CanAssertWebContentTimeFIlterCustom {
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsWebContent.testcase:296]: Hi @r4ddek, change the test name to `CanAssertWebContentTimeFIlterCustom`

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsEventAnalysis.testcase [hunk 6]
```diff
@@ -181,7 +181,6 @@ definition {
 	@description = "Feature ID: LRAC-4266 | Automation ID: LRAC-10307 | Test Summary: Total Chart, Unique Chart and Average Chart data shown"
 	@priority = "5"
 	test CheckTotalAndUniqueAndAverageChartDataIsShown {
-		property portal.upstream = "quarantine";
 
 		// AC Refactor: It is not possible to assert the chart because the data takes some time to appear.
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsEventAnalysis.testcase:185]: Hi @r4ddek, could you remove these lines?

**Comment by r4ddek** [comment]
ci:forward

**Comment by r4ddek** [comment]
ci:stop

**Pull request closed without merging**